### PR TITLE
Add Form section Pull Request

### PR DESCRIPTION
**Pull request by desorgui**
## What?
I've added a Contact form section for this project. 

## Why?
These changes complete the project requirements.

## How?
I used Html and CSS (Grid and Flexbox) to be able to place each element

## Testing?
I've added Linter for testing all new CSS codes.

## Anything Else?
 In the Contact form section, the display is correct just for the mobile version.

I used template 2 of the Microverse students projects to implement my portfolio
https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=48%3A27

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -233,5 +233,78 @@ <h1>About me</h1>
...
+							type="email"
+							id="email"
+							required
+							placeholder="Email Address"
+						/>
+					</li>
+					<li>
+						<textarea name="text" id="text" required maxlength="500">
+Enter text here</textarea
+						>
```
Comment (AmaduKamara) [inline index.html]: - [ ] Kindly make some changes to the form's `textarea` field. You can add the `placeholder` attribute to the `textarea` element and then place the message text there. The placeholder attribute is there to give a guide text to a user on what is to be entered in an input field.
- Hint 
```mdx
<textarea name="text" id="text" required maxlength="500" placeholder="Enter text here"></textarea>
```

- [ ] Kindly work on the styles for the icons below the form. Currently, you are not following the design guidelines on the Figma template given. Please fix the styles by removing the blue background of the icons.

**Your design**
image

**The Figma Design**
image

File: index.html [hunk 2]
```diff
@@ -233,5 +233,78 @@ <h1>About me</h1>
...
+		<section id="form-section">
+			<div id="form-text" class="form-text">
+				<p class="text">
+					I'm always interested in hearing about new projects, so if you'd like
+					to chat please get in touch.
+				</p>
+			</div>
+			<form
+				action="https://formspree.io/f/form"
+				id="form"
```
Comment (AmaduKamara) [inline index.html]: I can see you are following the structure of the form action URL here, 👍.

- [ ] Your form is not being submitted right now due to the wrong action URL given. Kindly create an account on the form spree website, then create a form on the account you created and copy the action URL of that account. It's that URL that you need to pass on the form's action attribute in your project. Please fix that 👍
- Below is the response from form spree
image

**Review by AmaduKamara — CHANGES_REQUESTED** [review]
# Status: Required Changes ♻️♻️♻️ 

Hi Team, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

AlonzoLeroneThisCloseGIF


## Highlights
✅ Correct Github flow
✅ No linter error
✅ README is okay

## Required Changes ♻️
_Check the comments under the review._

- [ ] Kindly check the inline comments and the screenshots placed in the comments to guide you on what needs to be fixed. 👍 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Review by ShahierNashaat — APPROVED** [review]
Hi @desorgui , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Approved :ok_hand: :ok_hand: :ok_hand:

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**